e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs

```diff
@@ -107,10 +113,10 @@ public void Dispose()
 
         string IDeviceConfig.DeviceId => GetStringOrDefault(TestConstants.EnvironmentVariablesNames.IOT_EDGE_DEVICE_ID,
             () => throw new Exception("IoT Edge device id is not provided."));
-
+      
```

Comment:
Remove trailing whitespace. Please install [Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer) VS Extension so that it automatically removes them.

Reply:
removed